lib/backbone-provider.js

```diff
@@ -1,25 +1,30 @@
-const { Component, Children } = require('react');
+const React = require('react');
+const {Children, Component} = React;
 const PropTypes = require('prop-types');
+const ConnectBackboneToReactContext = require('./context.js'); // eslint-disable-line no-unused-vars
 
 class BackboneProvider extends Component {
-  getChildContext() {
-    return {
-      models: this.props.models,
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      models: props.models,
     };
   }
 
```

Comment:
We need to implement `getDerivedStateFromProps` to copy over new prop `models` when they are given.

Or I wonder if we could just use `this.props` as the `value` given to the provider.